topbeat/beater/topbeat.go

```diff
@@ -41,34 +42,46 @@ func (tb *Topbeat) Config(b *beat.Beat) error {
 		return err
 	}
 
-	if tb.TbConfig.Input.Period != nil {
-		tb.period = time.Duration(*tb.TbConfig.Input.Period) * time.Second
+	if tb.TbConfig.Topbeat != nil && tb.TbConfig.Input != nil {
+		return fmt.Errorf("topbeat and input are set in config. Only one can be enabled. input is deprecated, use topbeat.")
+	}
+
+	// Copy shipper to config to beat @deprecated
+	if tb.TbConfig.Input != nil {
+		logp.Warn("input config is deprecated. Please use beat topbeat.")
```

Comment:
This could be a bit clearer, sice both input and topbeat have overloaded meaning. Perhaps: "The 'input' configuration section is deprecated. Please use 'topbeat' instead".